include/alp/algorithms/cholesky.hpp

```diff
@@ -50,8 +50,8 @@ namespace alp {
 			std::enable_if_t<
 				is_matrix< MatL >::value &&
 				is_matrix< MatH >::value &&
-				structures::is_a< typename  MatL::structure, structures::UpperTriangular >::value &&
-				structures::is_a< typename  MatH::structure, structures::Symmetric >::value &&
+				structures::is_a< typename MatL::structure, structures::UpperTriangular >::value &&
+				structures::is_a< typename MatH::structure, structures::Symmetric >::value &&
```

Comment:
We do have a `structures::SymmetricPositiveDefinite` that we should use here (and also elsewhere below). Not sure if that would work out of the box though. In case we should make a not to update this.